### PR TITLE
Fix export dropdown dropdown options

### PR DESCRIPTION
**Pull request by atjeff**
We noticed that clicking "All rows" only selected the current rows, and "Current rows" selected all of the rows for export / printing. This seems to be the bug here

### CODE REVIEW

**Comment by atjeff** [comment]
I tried to add a unit test:
```ts
describe('exportTo', () => {
        let tubularInstance: ITbInstance;

        beforeEach(async () => {
            tubularInstance = useTubular(columns, localData);

            await waitForDomChange();
        });

        it('should call getAllRecords if allRows is true', async () => {
            const getLocalDataSourceSpy = jest.spyOn(helpers, 'getLocalDataSource');

            tubularInstance.api.exportTo(true, jest.fn());

            expect(getLocalDataSourceSpy).toHaveBeenCalled();
        });

        it('should call getAllRecords if allRows is false', async () => {
            const getLocalDataSourceSpy = jest.spyOn(helpers, 'getLocalDataSource');

            tubularInstance.api.exportTo(false, jest.fn());

            expect(getLocalDataSourceSpy).not.toHaveBeenCalled();
        });
    });
``` but I guess you can't use hooks outside of a component, so not sure how you would test that api.

**Comment by kadosh** [comment]
hi @atjeff ,

In order to test hooks you need to have a component. That's the recommended way to proceed with it. Let me work on that.

**Comment by geoperez** [comment]
This PR has some conflicts.

**Comment by atjeff** [comment]
@geoperez Yes, it's been open a while and it seems you guys have since upgraded tubular-common, causing some conflicts. Is @kadosh Still doing to work on a test or should I fix this conflict?

**Comment by kadosh** [comment]
Hey @atjeff , I've been very busy. Could you please fix the conflicts and we can revisit the test in a separate way. I'm considering moving some of the logic in the hook into a helper function so that we can test it directly.

**Comment by atjeff** [comment]
Sure!

File: src/useTubular.ts [hunk 1]
```diff
@@ -3,18 +3,18 @@ import * as ReactDom from 'react-dom';
...
     CompareOperators,
+    DataGridStorage,
     GridRequest,
     GridResponse,
-    DataGridStorage,
-    TubularHttpClientAbstract,
     LocalStorage,
     NullStorage,
     sortColumnArray,
+    TubularHttpClientAbstract,
```
Comment (atjeff) [inline src/useTubular.ts]: This import sorting is from the vscode setting
```
"editor.codeActionsOnSave": {
        "source.organizeImports": true
    },
```

Which I believe comes from the ts compiler

Comment (kadosh) [reply src/useTubular.ts]: Yeah, but the error with lint might be not related to this one.

**Comment by geoperez** [comment]
Please ignore the LINT Action, I need to check what is going on.

**Comment by atjeff** [comment]
I didn't get any output for `npm run lint`

**Comment by kadosh** [comment]
Thank you @atjeff

**Pull request merged**